mds/server/settings.py

```diff
@@ -154,15 +152,4 @@
 
 # Poller configuration
 POLLER_TOKEN_CACHE = "default"
-
-POLLER_TOKEN_ENCRYPTION_KEY = CONFIG.getstr(
-    "poller.token_encryption_key",
-    # You obviously need to change it in production! I need one for tests
-    "8DhMPDYWFewVYz5m-zfwv4ebx4p4pF6-GvFQz8AOiRA=",
-).encode(  # Must be bytes
-    "ascii"
-)
-if len(base64.urlsafe_b64decode(POLLER_TOKEN_ENCRYPTION_KEY)) != 32:
-    raise ImproperlyConfigured(
-        "The encryption key must be a URL-safe base64-encoded 32-byte key."
-    )
+POLLER_TOKEN_ENCRYPTION_KEY = fernet.Fernet.generate_key()  # Reset on each restart
```

Comment:
Are you sure it won't be recomputed on each thread this way ?

Reply:
From what I remember, the whole optimization pass from Raphaël made it so that the application is forked after the settings are loaded, to avoid reloading them each time. The bootstrap is polling a ping view or something to know when the wsgi app is ready.

Reply:
If this is all too related to our infrastructure, I was also considering leaving this alone, and make this change in plato.

Reply:
Let's try this here, it seems KISS to me like this. Just keep an eye on this on deployment to check if this goes well.